src/SSOToken.php

```diff
@@ -183,12 +128,42 @@ protected function getClaim($claim) {
 	 */
 	protected function getAllClaims() {
 
-		$res = [];
-		$claims = $this->token->getClaims();
+		return $this->token->claims()->all();
+	}
+
+	/**
+	 * Decides between the new key methods, the JWT library offers
+	 *
+	 * @param string $appSecret
+	 * @return Key
+	 */
+	private function getKey(string $appSecret) {
+		if(strpos($appSecret,'-----') === 0 ) {
+			$key = InMemory::plainText($appSecret);
+		} else if (strpos($appSecret, 'file://') == 0 ) {
+			$key = InMemory::file($appSecret);
+		} else {
+			$key = InMemory::base64Encoded($appSecret);
+		}
+		return $key;
+	}
 
-		foreach($claims as $claim)
-			$res[$claim->getName()] = $claim->getValue();
+	/**
+	 * Formats the leeway integer value into a DateInterval
+	 *
+	 * @param int $leeway
```

Comment:
A description of what type and format is expected for this parameter would be helpful (i.E. leeway interval in seconds)